virttest/qemu_vm.py

```diff
@@ -2186,6 +2186,7 @@ def sort_key(dev):
             boot_opts["once"] = params.get("boot_once")
             boot_opts["strict"] = params.get("boot_strict")
             boot_opts["reboot-timeout"] = params.get("boot_reboot_timeout")
+            boot_opts["splash-time"] = params.get("splash_time")
```

Comment:
`boot_splash_time` please, to avoid possible name conflict. And do we consider add the option 'splash' together? Please ignore it if we have no such test requirement.

Reply:
Updated. No such test requirement, many thanks.